src/backends/tensorrt/tensorrtlib.cc

```diff
@@ -417,7 +417,32 @@ namespace dd
         _net_mutex); // no concurrent calls since the net is not
                      // re-instantiated
 
-    auto predict_dto = ad.createSharedDTO<DTO::ServicePredict>();
+    oatpp::Object<DTO::ServicePredict> predict_dto;
+    if (ad.has("dto"))
+      {
+        // cast to ServicePredict...
+        auto any = ad.get("dto").get<oatpp::Any>();
```

Comment:
This looks a temporary bypass until the full pipeline is DTO. Maybe we should add a comment that says so ?